package.json

```diff
@@ -36,15 +37,18 @@
     "invariant": "^2.2.4",
     "optimist": "^0.6.1",
     "process": "^0.11.10",
-    "uuid": "^7.0.2"
+    "uuid": "^8.3.0"
```

Comment:
Could do it in this PR or not but node supports `crypto.randomBytes()`, no need for the uuid package. Then we'd generate only 22 random bytes instead of 32 as a microoptimization. (And for UUIDv4 strings there's `crypto.randomUUID()` now.)